### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by b4rdha**
PR template went missing after the refactor, adding it here now.

### CODE REVIEW

**Comment by LIYUHAN22** [comment]
Nice catch for the missing

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#[[issue Nr]] << Descriptive message >>
```
Comment (MajaLinder) [inline pull_request_template.md:1]: I'd suggest we have "closes #[[issue Nr]]" instead

Comment (LIYUHAN22) [reply pull_request_template.md:1]: > I'd suggest we have "closes #[[issue Nr]]" instead

I think maybe we will get the same result from these two code.

**Review by LIYUHAN22 — APPROVED** [review]
Nice catch for the missing

**Review by MajaLinder — APPROVED** [review]

**Pull request merged**